src/main/kotlin/app/batch/HbaseClient.kt

```diff
@@ -0,0 +1,84 @@
+package app.batch
+
+import org.apache.hadoop.hbase.HBaseConfiguration
+import org.apache.hadoop.hbase.TableName
+import org.apache.hadoop.hbase.client.*
+import org.apache.hadoop.hbase.io.TimeRange
+import org.apache.hadoop.hbase.util.Bytes
+
+open class HbaseClient(
+        val connection: Connection,
```

Comment:
I can't find unit tests for this class. Should be some you can take from HTME.

Reply:
There is no business logic in this class except interaction with hbase client library  same as Kafka to Hbase, I believe in HTME it is HbaseReader which  has business logic

Reply:
I would argue you could be calling the wrong methods in the hbase client and/or passing the wrong things, so you want to unit test your interactions are right, but if there are no HTME tests then happy for them to be left here as given the integration tests the risk is low.